### PR TITLE
Bl2782 migration dropping size and orientation

### PR DESCRIPTION
**Pull request by hatton**
Fix BL-2782 Migration to custom pages was dropping the size & layout

I added a couple tests for this and a related future problem, which is that of dropping other classes that are on the page. now, it merges the two classes, but drops the now irrelevant classes like "imageOnTop".

### CODE REVIEW

File: src/BloomExe/MiscUI/BloomIntegrityDialog.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public BloomIntegrityDialog()
 		public static bool CheckIntegrity()
 		{
 			var errors = new StringBuilder();
-			var files = new[] { "Bloom.chm", "PdfDroplet.exe", "Chorus.exe", "GeckofxHtmlToPdf.exe", "BloomBookUploader.exe", "optipng.exe" };
+			var files = new[] { "Bloom.chm", "PdfDroplet.exe", "Chorus.exe", "GeckofxHtmlToPdf.exe", "optipng.exe" };
```
Comment (StephenMcConnel) [inline src/BloomExe/MiscUI/BloomIntegrityDialog.cs]: Is this just another change in passing that you decided needed to be done?

File: src/BloomTests/Book/HtmlDomTests.cs [hunk 2]
```diff
@@ -157,5 +158,54 @@ public void SortStyleSheetLinks_LeavesOverridesAtEndAndSpecialFilesInMiddle()
...
+			AssertHasClases("one two", MergeClasses("dropMe dropMe dropMe ", "one two", new[] { "dropMe" }));
+		}
+		[Test]
+		public void MergeClassesIntoNewPage_MergesAndDropsItemsInDropList()
+		{
+			AssertHasClases("one two three", MergeClasses("one drop two delete", "three", new[] { "drop","delete" }));
+		}
+
+		private void AssertHasClases(string expectedString, string actualString)
+		{
```
Comment (StephenMcConnel) [inline src/BloomTests/Book/HtmlDomTests.cs]: The method AssertHasClases should be AssertHasClasses, but I'm willing to let it pass.

**Comment by StephenMcConnel** [comment]
Generally looks okay.  One unrelated change and one annoying typo in a method name, but nothing that absolutely prevents this from being checked in (assuming the unrelated change is intentional).

File: src/BloomExe/MiscUI/BloomIntegrityDialog.cs (hunk 1 above)
Comment (hatton) [inline src/BloomExe/MiscUI/BloomIntegrityDialog.cs]: Yes. It has its own commit with a message.

File: src/BloomTests/Book/HtmlDomTests.cs (hunk 2 above)
Comment (hatton) [inline src/BloomTests/Book/HtmlDomTests.cs]: oops. Will fix

**Pull request merged**